src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -214,6 +214,11 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
       @Override
       void afterResolve(ResolvableDependencies resolvableDependencies) {}
     })
+
+    project.tasks.withType(AbstractCompile).all { AbstractCompile compile ->
```

Comment:
Could you please use `configureEach` instead of `all`?
`all` results in eager task realization, while `configureEach` supports [configuration avoidance](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html)

Reply:
Thanks for catching this! Resolved in #115.